tests/commands/gen-js/__snapshots__/index.js

```diff
@@ -33,19 +33,7 @@ export default class Analytics {
         var errs__0 = errors;
         var valid1 = true;
         var data1 = data.properties;
-        if (data1 === undefined) {
-          valid1 = false;
-          var err = {
-            keyword: "required",
-            dataPath: (dataPath || "") + "",
-            schemaPath: "#/required",
-            params: { missingProperty: "properties" },
-            message: "should have required property 'properties'"
-          };
-          if (vErrors === null) vErrors = [err];
-          else vErrors.push(err);
-          errors++;
-        } else {
+        if (data1 !== undefined) {
```

Comment:
Couldn't find a change in the fixture that would lead to this getting stripped out 😕

Reply:
Good eye! But it's because `properties` is no longer required. If you have a Tracking Plan event with no properties, then it no longer adds `properties` as a required field.

https://github.com/segmentio/typewriter/pull/34/files#diff-b0dfe8b15729c6e2e68b53f64407622eL18